code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -79,15 +80,12 @@
 	item_state = "clown_shoes"
 	slowdown = SHOES_SLOWDOWN+1
 	item_color = "clown"
-	var/footstep = 1	//used for squeeks whilst walking
 	pockets = /obj/item/storage/internal/pocket/shoes/clown
+	var/list/step_sounds = list('sound/effects/clownstep1.ogg'=1,'sound/effects/clownstep2.ogg'=1)
```

Comment:
the step sounds should live only in the datum, not on the shoes themself I think

Reply:
I'll switch this to the other option which is to only do this work in initialize, but the only option other than that is to have a different component subtype for anything that wants a different squeak sound.

Reply:
hey you're the expert on this